duos-api/src/Dtos/userDto.ts

```diff
@@ -0,0 +1,12 @@
+import { ValidateNested } from 'class-validator';
```

Comment:
(Optional) You can move this dto folder into the user folder, and going forward just have a dto folder within each folder like user. Then you could also add a index.ts file inside dto folder with ``` export * from './userDto.ts';``` to make it easier to import.

Reply:
I'll implement this with the changes I need to make in a separate branch